rust/tools/authorization-logic/test_inputs/negation

```diff
@@ -0,0 +1,8 @@
+"TestPrin" says foo("x") :- !bar("x").
+"TestPrin" says baz("x") :- bar("x").
+
+// yes
+q1 = query "TestPrin" says foo("x")?
+// no
+q2 = query "TestPrin" says baz("x")?
+
```

Comment:
In this test, `q1` should be true because `bar` does not have `"x"` (or indeed, anything) in it, right? This test would be a bit better controlled if we also checked that it did the right thing for a symbol that was present in a relation, either as an extension to this test or a separate one.

Reply:
ok fixed.